ui/src/components/customModal/ResetPwdModal.tsx

```diff
@@ -0,0 +1,84 @@
+import { access } from '@/api';
+import type { ParamResetPasswordParam } from '@/api/generated';
+import { encryptText, usePublicKey } from '@/hook/usePublicKey';
+import { Form, Input, message } from 'antd';
+import CustomModal from '.';
+
+interface ResetPwdModalProps {
+  visible: boolean;
+  setVisible: (visible: boolean) => void;
+  successCallback?: () => void;
+}
+
+export default function ResetPwdModal({
+  visible,
+  setVisible,
+  successCallback,
+}: ResetPwdModalProps) {
+  const [form] = Form.useForm();
+  const publicKey = usePublicKey();
+  const handleSubmit = async () => {
+    try {
+      await form.validateFields();
+      form.submit();
+    } catch (err) {}
+  };
+  const onFinish = async (values: ParamResetPasswordParam) => {
+    values.oldPassword = encryptText(values.oldPassword!, publicKey) as string;
+    values.password = encryptText(values.password!, publicKey) as string;
+    const res = await access.resetPassword(values);
+    if (res.successful) {
+      message.success('操作成功！');
+      if (successCallback) successCallback();
+      form.resetFields();
+      setVisible(false);
+    }
+  };
+  return (
+    <CustomModal
+      title="重置密码"
+      isOpen={visible}
+      handleOk={handleSubmit}
+      handleCancel={() => {
+        form.resetFields();
+        setVisible(false);
+      }}
+    >
+      <Form form={form} onFinish={onFinish}>
+        <Form.Item
+          rules={[
+            {
+              required: true,
+              message: '请输入',
+            },
+          ]}
+          label="原密码"
+          name={'oldPassword'}
+        >
+          <Input type="password" placeholder="请输入" />
+        </Form.Item>
+        <Form.Item
+          rules={[
+            {
+              required: true,
+              message: '请输入',
+            },
+            ({ getFieldValue }) => ({
+              validator(_, value) {
+                const oldPwd = getFieldValue('oldPassword');
+                if (oldPwd === value) {
+                  return Promise.reject('新密码不能和原密码相同');
+                }
+                return Promise.resolve();
+              },
+            }),
+          ]}
+          label="新密码"
+          name={'password'}
+        >
+          <Input type="password" placeholder="请输入" />
+        </Form.Item>
```

Comment:
There should be an another input named like `PasswordConfirm` for confirming password that the user inputs.

Reply:
Here is to change the password, `PasswordConfirm` is in the `./pages/ResetPwd` directory

Reply:
Yes I know. User needs to confirm the password he input just now.

Reply:
okey.